escrow/payment_handler.go

```diff
@@ -93,7 +96,39 @@ func (h *paymentChannelPaymentHandler) getPaymentFromContext(context *handler.Gr
 }
 
 func (h *paymentChannelPaymentHandler) Complete(payment handler.Payment) (err *handler.GrpcError) {
-	return paymentErrorToGrpcError(payment.(*paymentTransaction).Commit())
+	if err = paymentErrorToGrpcError(payment.(*paymentTransaction).Commit()); err != nil {
+		PublishChannelStats(payment)
```

Comment:
Two options here: to fail or to NOT fail client call if metrics were not published. I see the second one is implemented. At the same time in `PublishChannelStats` you return `GrpcError`. Did you implement the second option by intention?

Reply:
We are looking at building a Swagger UI to do this for Market place service , work in progress

Reply:
in the cucrrent state , for now we dont want to fail any client calls when PublishChannelStats fails, however I wanted to keep this option , just in case we need to manage this error in to the future and also have some testable code to see the error when an Error happens